tests/ecmp/inner_hashing/test_wr_inner_hashing.py

```diff
@@ -17,62 +18,65 @@
-        timestamp = datetime.now().strftime('%Y-%m-%d-%H:%M:%S')
-        log_file = "/tmp/wr_inner_hash_test.DynamicInnerHashTest.{}.{}.{}.log".format(outer_ipver, inner_ipver, timestamp)
-        logging.info("PTF log file: %s" % log_file)
-
-        # to reduce test run time, check one of encapsulation formats
-        outer_encap_format = random.choice(OUTER_ENCAP_FORMATS).split()
-        logging.info("Tested encapsulation format: {}".format(outer_encap_format[0]))
-
-        outer_src_ip_range, outer_dst_ip_range = get_src_dst_ip_range(outer_ipver)
-        inner_src_ip_range, inner_dst_ip_range = get_src_dst_ip_range(inner_ipver)
-
-        balancing_test_times = 200
-        balancing_range = 0.3
-
-        duthost.command('sudo config save -y')
-        reboot_thr = threading.Thread(target=reboot, args=(duthost, localhost, 'warm',))
-        reboot_thr.start()
-
-        ptf_runner(ptfhost,
-                   "ptftests",
-                   "inner_hash_test.InnerHashTest",
-                   platform_dir="ptftests",
-                   params={"fib_info": FIB_INFO_FILE_DST,
-                           "router_mac": router_mac,
-                           "src_ports": vlan_ptf_ports,
-                           "hash_keys": hash_keys,
-                           "vxlan_port": VXLAN_PORT,
-                           "inner_src_ip_range": ",".join(inner_src_ip_range),
-                           "inner_dst_ip_range": ",".join(inner_dst_ip_range),
-                           "outer_src_ip_range": ",".join(outer_src_ip_range),
-                           "outer_dst_ip_range": ",".join(outer_dst_ip_range),
-                           "balancing_test_times": balancing_test_times,
-                           "balancing_range": balancing_range,
-                           "outer_encap_formats": outer_encap_format,
-                           "symmetric_hashing": symmetric_hashing},
-                   log_file=log_file,
-                   qlen=PTF_QLEN,
-                   socket_recv_size=16384)
-        reboot_thr.join()
+        with allure.step('Run ptf test InnerHashTest and warm-reboot in parallel'):
+            timestamp = datetime.now().strftime('%Y-%m-%d-%H:%M:%S')
+            log_file = "/tmp/wr_inner_hash_test.DynamicInnerHashTest.{}.{}.{}.log".format(outer_ipver, inner_ipver, timestamp)
+            logging.info("PTF log file: %s" % log_file)
+
+            # to reduce test run time, check one of encapsulation formats
+            outer_encap_format = random.choice(OUTER_ENCAP_FORMATS).split()
+            logging.info("Tested encapsulation format: {}".format(outer_encap_format[0]))
+
+            outer_src_ip_range, outer_dst_ip_range = get_src_dst_ip_range(outer_ipver)
+            inner_src_ip_range, inner_dst_ip_range = get_src_dst_ip_range(inner_ipver)
+
+            balancing_test_times = 200
+            balancing_range = 0.3
+
+            duthost.command('sudo config save -y')
```

Comment:
Given we do config save here, and we modify device config in the test is there a possibility that we overwrite the base config with something different which will be persisted beyond the test?